xml/ay_bootloader.xml

```diff
@@ -102,352 +102,284 @@
 <screen>&lt;boot_mbr&gt;false&lt;/boot_mbr&gt;</screen>
-        </entry>
-       </row>
-       <row>
-        <entry>
-         <para>
-          <literal>boot_root</literal>
-         </para>
-        </entry>
-        <entry>
-         <para>
-          Write &grub; to <filename>/</filename> partition.
-         </para>
+      </listitem>
+     </varlistentry>
+     
+     <varlistentry>
+      <term>boot_root</term>
+      <listitem>
+       <para>
+        Write &grub; to <filename>/</filename> partition.
+       </para>
 <screen>&lt;boot_root&gt;false&lt;/boot_root&gt;</screen>
-        </entry>
-       </row>
-       <row>
-        <entry>
-         <para>
-          <literal>generic_mbr</literal>
-         </para>
-        </entry>
-        <entry>
-         <para>
-          Write generic boot code to the MBR (will be ignored if <literal>boot_mbr</literal>
-          is set to <literal>true</literal>).
-         </para>
+      </listitem>
+     </varlistentry>
+     
+     <varlistentry>
+      <term>generic_mbr</term>
+      <listitem>
+       <para>
+        Write generic boot code to the MBR (will be ignored if
+        <literal>boot_mbr</literal> is set to <literal>true</literal>).
+       </para>
 <screen>&lt;generic_mbr config:type="boolean"&gt;false&lt;/generic_mbr&gt;</screen>
-        </entry>
-       </row>
-       <row>
-        <entry>
-         <para>
-          <literal>gfxmode</literal>
-         </para>
-        </entry>
-        <entry>
-         <para>
-          Graphical resolution of the &grub; screen (requires
-          &lt;terminal&gt; to be set to <literal>gfxterm</literal>.
-          Valid entries are <literal>auto</literal>,
-          <literal><replaceable>HORIZONTAL</replaceable>x<replaceable>VERTICAL</replaceable></literal>,
-          or
-          <literal><replaceable>HORIZONTAL</replaceable>x<replaceable>VERTICAL</replaceable>x<replaceable>COLOR
-          DEPTH</replaceable></literal>. You can see the screen
-          resolutions supported by &grub; on a particular system by
-          using the <command>vbeinfo</command> command at the &grub; command line in
-          the running system.
-         </para>
+      </listitem>
+     </varlistentry>
+     
+     <varlistentry>
+      <term>gfxmode</term>
+      <listitem>
+       <para>
+        Graphical resolution of the &grub; screen (requires |&lt;terminal&gt; to
+        be set to <literal>gfxterm</literal>.
+       </para>
+       <para>
+        Valid entries are <literal>auto</literal>, <literal><replaceable>
+        HORIZONTAL</replaceable>x<replaceable>VERTICAL</replaceable></literal>,
+        or <literal><replaceable>HORIZONTAL</replaceable>x<replaceable>VERTICAL
+        </replaceable>x<replaceable>COLOR DEPTH</replaceable></literal>. You can
+        see the screen resolutions supported by &grub; on a particular system by
+        using the <command>vbeinfo</command> command at the &grub; command line
+        in the running system.
```

Comment:
vbeinfo is not in suse. xrandr works.

Reply:
> xrandr works.

Snag is, that requires X.11, I think.

But this is a good catch.

From the shell, there is `hwinfo --vbe` but I think this is a GRUB-EFI issue.

I found a suggestion of `videoinfo` but I do not have a UEFI machine to test. Do you?

Reply:
Oh never mind, "using the vbeinfo command at the GRUB 2 command line in the running system." Ignore.